integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/AwsSqsSteps.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.awaitility.Awaitility;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.smartevents.integration.tests.resources.AwsSqsResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+public class AwsSqsSteps {
+
+    private TestContext context;
+
+    public AwsSqsSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @And("^create an SQS queue on AWS called \"([^\"]*)\" within (\\d+) (?:minute|minutes)$")
+    public void createQueue(String queueName, int timeoutMinutes) {
+        Awaitility.await()
```

Comment:
Can you provide a generation of unique SQS name? That way we can make sure that parallel execution of the same test won't interfere. See for example Kafka approach - https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/TestContext.java#L84-L93

Reply:
done